docs/api.md

```diff
@@ -526,3 +527,66 @@ coll.use(new TitleCharTransformer("?"));
 There's currently no way to deal with adding tranformers to an already filled remote database; that would mean remote data migrations, and both Kinto and Kinto.js don't provide this feature just yet.
 
 **As a rule of thumb, you should only start using transformers on an empty remote collection.**
+
+## Id schemas
+
+By default, kinto.js uses UUID4 strings for record id's. If you want to work with an existing body of data, this may not be what you want.
+
+You can define a custom id schema on a collection by calling the `Collection#use()` method, which accepts a `Kinto.IdSchema`-derived object instance:
```

Comment:
I feel having one `use` API which accepts different types of input is misleading. I've seen this pattern used for express (the node.js framework) in the past and it seems to be counter intuitive (e.g. really easy to use, not as easy to read).

Instead, we could have different methods to *install* the different configurations.

Here are different proposals:
- Have a `Collection#configure` to which we would pass an object with an `id-schema` key;
- Variant, pass two arguments to `Collection#setConfig`. e.g. `setConfig('id-schema', object)`
- Have a `Collection#setIdSchema` to which we pass an object;

Thoughts?